PathOfBuildingCommunity/PathOfBuilding
# Fix Projectile Attack Mod Parsing

### Description of the problem being solved:

Some skills are tagged ProjectileAttack (and Attack) but not Projectile. This generally means the skill deals projectile attack damage, but does not actually fire a projectile that can collide with terrain (e.g. Rain of Arrows, Storm Rain, Blast Rain, Toxic Rain, Artillery Ballista). In order for nodes like Lethality to correctly match these skills (while still matching all normal projectile attacks) we need to check the ProjectileAttack tag explicitly.

### Steps taken to verify a working solution:
- Check DPS of Storm Rain with Lethality allocated in 2.13.0
- Update ModParser.lua and rebuild ModCache.lua
- Check DPS of Storm Rain with Lethality allocated again

### Link to a build that showcases this PR:
https://pastebin.com/ZeCpW2CU

### Before screenshot:
![](http://puu.sh/IyZ9p/c23ddd167d.png)

### After screenshot:
![](http://puu.sh/IyZ9X/a7d3cebace.png)
